pkg/cmd/cli/describe/describer.go

```diff
@@ -65,7 +66,7 @@ func (d *BuildDescriber) DescribeUser(out *tabwriter.Writer, label string, u bui
 	}
 }
 
-func (d *BuildDescriber) DescribeParameters(p buildapi.BuildParameters, out *tabwriter.Writer) {
+func (d *BuildDescriber) DescribeParameters(p buildapi.BuildParameters, namespace string, out *tabwriter.Writer) {
```

Comment:
you don't need this namespace arg if you rework the describer as i suggest below.